lib/vanagon/engine/base.rb

```diff
@@ -6,12 +6,14 @@ class Engine
     class Base
       attr_accessor :target, :remote_workdir, :name
 
-      def initialize(platform, target = nil)
+      def initialize(platform, target = nil, opts = {})
```

Comment:
Nitpick: Since Vanagon requires Ruby 2.1 or better, we chose not to use the option hash pattern. Instead we used [keyword arguments](https://robots.thoughtbot.com/ruby-2-keyword-arguments). Again, I won't block on it but we're probably going to fix it up immediately after merge.